### PR TITLE
Remove HackerSM64 discord link

### PR DESCRIPTION
**Pull request by aglab2**

### CODE REVIEW

**Comment by gheskett** [comment]
so true

**Comment by PyrolluxJay** [comment]
bubsyshatter

File: README.md [hunk 1]
```diff
@@ -7,8 +7,6 @@ When building from a freshly cloned repo, the baserom extractor will overwrite a
 
 **AFTER CLONING THE REPO, CHECK OUT THE `include/config` FOLDER BEFORE ANYTHING ELSE! THERE'S A LOT OF STUFF IN THIS REPO THAT CAN BE TOGGLED THERE.**
 
-HackerSM64 now has a discord server! https://discord.gg/brETAakcXr
-
```
Comment (gheskett) [inline README.md:11]: ~Actually, can you just delete the entire HackerSM64 section and just replace it with:~
~`Made with HackerSM64: https://github.com/HackerN64/HackerSM64`~
~Or similar~

EDIT: Better yet, can you just turn the BtCM and HackerSM64 credits into hyperlinks?

**Review by gheskett — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aglab2) [reply README.md:11]: There's already a PR that does that https://github.com/rovertronic/Mario-Builder-64/pull/31

Comment (gheskett) [reply README.md:11]: Didn't realize I cut out the part where I also mentioned scrapping the entirety of the HackerSM64 section, whoops. Either way, the Discord link is gone now.

**Pull request merged**